doc/attributes_mathMap.dox

```diff
@@ -2,7 +2,7 @@ namespace ast {
 /**
 @page mathmap_attributes MathMap Attributes
 
-@section MathMap_Attributes @ref MathMap Attributes
+@section MathMap_Attributes MathMap Attributes
```

Comment:
It seems strange for `@section` to render as `<h1>`. I'm used to reserving `<h1>` for the page title.